src/platform/utilities/api/index.js

```diff
@@ -83,6 +83,7 @@ export function apiRequest(resource, optionalSettings = {}, success, error) {
     .catch(err => {
       Sentry.withScope(scope => {
         scope.setExtra('error', err);
+        scope.setFingerprint(['{{default}}', scope._tags?.source]);
```

Comment:
All of the `vets_client_error` errors are rolled up into one Sentry group, which makes it impossible to find the errors specific to your app. This should separate them out by source, which is set to the bundle you're in (or unknown).

Reply:
Context for other folks looking at this: We have the ability to search through Sentry to find errors related to a single application. This is done using `tags`. I believe `startApp` is what sets the `source` tag for all errors thrown in an application.

![image](https://user-images.githubusercontent.com/12970166/74767356-94ba3200-523b-11ea-94f2-302db62f5761.png)
_Filter sentry events by source application_

If an issue has multiple events, each with a different `source` tag, they'll all be grouped together. The idea here is to ungroup them so there's a separate issue where each event in that issue has the same `source`.

Does that sound about right, @jbalboni ?

Reply:
Correct, and to be clear, it only applies to this particular error, not all errors. We could change the default fingerprint, but I'm not sure if that's a good idea or not.